authors/apps/articles/models.py

```diff
@@ -21,6 +22,10 @@ class Article(models.Model):
     author = models.ForeignKey(ProfileModel.Profile,
                                on_delete=models.CASCADE)
     body = models.TextField()
+    liked_by = models.ManyToManyField(to=settings.AUTH_USER_MODEL, related_name='liked_articles',
```

Comment:
Line lengths are still too long in some files. Could this be corrected?

Reply:
Just fixed a bunch of them
To minimize the number of rebase conflicts, I'll leave the pre-existing ones for later